server/handler/base.go

```diff
@@ -185,7 +185,7 @@ func (b *Base) EvaluateFetchedConfig(ctx context.Context, prctx pull.Context, cl
 
 	result := evaluator.Evaluate(ctx, prctx)
 	if result.Error != nil {
-		statusMessage := fmt.Sprintf("Error evaluating policy defined by %s", fetchedConfig)
+		statusMessage := fmt.Sprintf("Error evaluating policy defined by %s. This may be temporary. Create a PR comment to try again.", fetchedConfig)
```

Comment:
I'm not sure this will actually show up in a useful way in the GitHub UI... we'd have to see where the truncation happens. Originally, I was thinking this would be part of the error banner in the details view, rather than in the status message. That way, if someone clicked through to see why the check failed, there would be a tip on how to resolve it.

That also raises the point that the details view does a fresh evaluation but does not update status checks, so you can get into a situation where by the time you check why something failed, it's not failing anymore. Maybe having the details view update status is another way to help here?